src/core/src/core_logic/PatchAssessor.py

```diff
@@ -57,11 +62,13 @@ def start_assessment(self):
         self.composite_logger.log("\n\nGetting available patches...")
         self.package_manager.refresh_repo()
         self.status_handler.reset_assessment_data()
+        number_of_tries = 0
```

Comment:
non-blocker: retry_count

Reply:
Sure, I will take care of this in a separate PR. Completing this PR.